src/php/Lang/Collections/Map/HashCollisionNode.php

```diff
@@ -16,19 +16,18 @@
  *
  * @implements HashMapNodeInterface<K, V>
  */
-class HashCollisionNode implements HashMapNodeInterface
+final class HashCollisionNode implements HashMapNodeInterface
 {
-    /** @var array<int, K|V> */
-    private array $objects;
-
+    /**
+     * @param array{K,V,K,V} $objects
+     */
```

Comment:
This one is (only) created in the `IndexedNode` class as follows, but feel free to change it in case any other PHPDoc works better 😄 

<img width="1103" alt="image" src="https://github.com/phel-lang/phel-lang/assets/6381924/ddb49a2c-e549-4845-be3d-d4cdb0ccf757">